VAMobile/e2e/tests/ContactInformation.e2e.ts

```diff
@@ -69,136 +103,167 @@ describe('Contact Info Screen', () => {
+    if(type === 'home phone' || type == 'work phone'){
+      await expect(element(by.text('Add your ' + type + ' number'))).toExist()
+    } else if (type === 'mobile phone') {
+      await expect(element(by.text('Add your cell phone number'))).toExist()
+    } else {
+      await expect(element(by.text('Add your ' + type))).toExist()
+    }
   })
+}
 
-  it ('should open the mobile phone number', async () => {
-    await device.launchApp({newInstance: true})
-    await loginToDemoMode()
-    await openProfile()
-    await openContactInfo()
-    await element(by.id('ContactInfoTestID')).scrollTo('bottom')
-    await element(by.id(ContactInfoE2eIdConstants.MOBILE_PHONE_ID)).tap()
-  })
+beforeAll(async () => {
+  await loginToDemoMode()
+  await openProfile()
+  await openContactInfo()
+})
 
-  it('should update the mobile phone with an extension', async () => {
-    await element(by.id(ContactInfoE2eIdConstants.PHONE_NUMBER_ID)).replaceText('276-608-6180')
-    await element(by.id(ContactInfoE2eIdConstants.PHONE_NUMBER_EXTENSION_ID)).replaceText('1234')
-    await element(by.text('Save')).tap()
-    await expect(element(by.text('Mobile phone saved'))).toExist()
-    await element(by.text('Dismiss')).tap()
-  })
+describe('Contact Info Screen', () => { 	
+  it('should match the Contact Info page design', async () => {
+		await expect(element(by.id(ContactInfoE2eIdConstants.MAILING_ADDRESS_ID))).toExist()
+		await expect(element(by.id(ContactInfoE2eIdConstants.HOME_ADDRESS_ID))).toExist()
+    await expect(element(by.id(ContactInfoE2eIdConstants.HOME_PHONE_ID))).toExist()
+		await expect(element(by.id(ContactInfoE2eIdConstants.WORK_PHONE_ID))).toExist()
+    await expect(element(by.id(ContactInfoE2eIdConstants.MOBILE_PHONE_ID))).toExist()
+		await expect(element(by.id(ContactInfoE2eIdConstants.EMAIL_ADDRESS_ID))).toExist()
+		await expect(element(by.text(ContactInfoE2eIdConstants.HOW_WE_USE_TEXT))).toExist()
+	}) 
 
-  it('should update the mobile phone number and remove the extension', async () => {
-    await element(by.id('ContactInfoTestID')).scrollTo('bottom')
-    await element(by.id(ContactInfoE2eIdConstants.MOBILE_PHONE_ID)).tap()
-    await element(by.id(ContactInfoE2eIdConstants.PHONE_NUMBER_EXTENSION_ID)).clearText()
-    await element(by.text('Save')).tap()
-    await expect(element(by.text('Mobile phone saved'))).toExist()
-    await element(by.text('Dismiss')).tap()
+  it('should tap on the how we use your contact information link and verify that it opens', async () => {
+    await expect(element(by.id('howWeUseContactInfoLinkTestID'))).toExist()
+    await element(by.id('howWeUseContactInfoLinkTestID')).tap()
+    await expect(element(by.text('How we use your contact information')).atIndex(0)).toExist()
+    await element(by.text('Close')).atIndex(0).tap()
   })
 
+  validateAddresses(ContactInfoE2eIdConstants.MAILING_ADDRESS_ID, 'Mailing')
+  validateAddresses('Home address Add your home address', 'Home')

```

Comment:
Could use the ID here? validateAddresses(ContactInfoE2eIdConstants.HOME_ADDRESS_ID, 'Home')

Reply:
swapped it in